docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -878,15 +854,11 @@ desired={<Twelve viewBox="140 40 680 500" />}
 
 <div>
 
-#### Current state
-
-Camunda 8 is reachable to the end-user but not yet exporting any data.
-
-#### Desired state
-
-You are initializing a new exporter to the recreated region. This will ensure that both Elasticsearch instances are populated, resulting in data redundancy.
-
-Separating this step from resuming the exporters is essential as the initialization is an asynchronous procedure, and you must ensure it's finished before resuming the exporters.
+| **Details**                 | **Current State**                                   | **Desired State**                                                                                     |
+| --------------------------- | --------------------------------------------------- | ----------------------------------------------------------------------------------------------------- |
+| **Camunda 8 Accessibility** | Reachable to end-users, but not exporting any data. | Start a new exporter to the recreated region.                                                         |
+| **Data Exporting**          | No data export currently in progress.               | Ensure that both Elasticsearch instances are populated for data redundancy.                           |
```

Comment:
I'd remove this step, it's just repeating what has been said already.
Add `Ensure that both Elasticsearch instances are populated for data redundancy. ` to the sentence above.

On it's own doesn't make sense. Who is ensuring that? The start of a new exporter.